nixos/modules/programs/foot/default.nix

```diff
@@ -35,24 +35,49 @@ in
     package = lib.mkPackageOption pkgs "foot" { };
 
     settings = lib.mkOption {
-      inherit (settingsFormat) type;
+      type = lib.types.submodule {
+        freeformType = settingsFormat.type;
+
+        options = {
+          main = lib.mkOption {
+            type = settingsFormat.type.nestedTypes.elemType;
+            default = { };
+            description = ''
+              Main configuration section (global section). All options that
+              could occur outside any section in {manpage}`foot.ini(5)` need to
+              be placed in this section.
+            '';
+          };
+        };
+      };
+
       default = { };
-      description = ''
-        Configuration for foot terminal emulator. Further information can be found in {command}`man 5 foot.ini`.
 
-        Global configuration has to be written under the [main] section.
+      description = ''
+        Configuration for foot terminal emulator. Further information can be
+        found in {manpage}`foot.ini(5)`.
```

Comment:
I tried `{manpage}` before I suggested using `{command}`.  Because the manual of foot.ini is not in [manpage-urls.json](https://github.com/NixOS/nixpkgs/blob/master/doc/manpage-urls.json), the rendered result of `{manpage}` is worse.  Here is the [doc] for `{manpage}`.

[doc]: https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#roles